specification/consumption/resource-manager/readme.md

```diff
@@ -40,13 +40,13 @@ input-file:
 ```
 
 ---
-### Tag: package-2018-01
```

Comment:
Is there still a January version of this, or did shipping schedule change?

Reply:
Good catch! It was just overwritten. I have added the January version back.